vpc/proto/vpc.proto

```diff
@@ -157,6 +159,13 @@ message AssignIPResponseV3 {
         // burst is the HTB class's ceil in bits per second
         uint64 burst = 2;
     }
+    message TransitionAssignment {
+        string assignmentId = 1;
+        // TODO: Add mark.
```

Comment:
What kind of mark?

Reply:
I decided the best way to handle bandwidth limiting would be to set SO_MARK on the sockets. The TSA would need to know on a container-by-container basis the mark to use when it injects the socket. 

This is a longer-term: TODO.